src/commcare_cloud/manage_commcare_cloud/test_environments.py

```diff
@@ -0,0 +1,12 @@
+from __future__ import print_function
+from commcare_cloud.commands.command_base import CommandBase
+
+
+class TestEnvironments(CommandBase):
+    command = 'test-environments'
+    help = "Run test environments"
+
+    def run(self, args, unknown_args):
+        import nose
+        nose.runmodule('commcare_cloud.manage_commcare_cloud',
+                       argv=['manage-commcare-cloud', '-v'])
```

Comment:
what does this actually do?

Reply:
Part of the point of this PR is to separate the tests that are based on the dimagi `environments/` file from everything else. This line is the from-python way of doing the equivalent of running

```
nosetests commcare_cloud.manage_commcare_cloud -v
```
from the command line. (Since tests themselves now live in `src/commcare_cloud/manage_commcare_cloud/tests/`)

Reply:
:+1: that's what I thought but was a bit confused for some reason.